runtime/shibuya/src/lib.rs

```diff
@@ -683,6 +695,8 @@ impl pallet_contracts::Config for Runtime {
     type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
     type MaxCodeLen = ConstU32<{ 128 * 1024 }>;
     type MaxStorageKeyLen = ConstU32<128>;
+    type UnsafeUnstableInterface = ConstBool<false>;
```

Comment:
Not sure this should be disabled on testnet, let's confirm with wasm team. CC @PierreOssun